extensions/estonian/src/extension.ts

```diff
@@ -0,0 +1,57 @@
+'use strict';
+// The module 'vscode' contains the VS Code extensibility API
+// Import the module and reference it with the alias vscode in your code below
+import * as vscode from 'vscode';
+interface CodeSpellCheckerExtension {
+    registerConfig(path: string): Promise<void>;
+    enableLocal(isGlobal: boolean, locale: string): Promise<void>;
+    disableLocal(isGlobal: boolean, locale: string): Promise<void>;
+}
+
+// 
+const locale = 'ee';
```

Comment:
I believe this should be `et` instead of `ee`. See: [Alpha-2 Code (et) Search Result - Codes for the representation of names of languages (Library of Congress)](https://www.loc.gov/standards/iso639-2/php/langcodes_name.php?iso_639_1=et)